Documentation/HowToGuides/AddingACollisionIgnorer/README.md

```diff
@@ -0,0 +1,64 @@
+# Adding A Collision Ignorer
+
+> * Level: Beginner
+>
+> * Reading Time: 5 minutes
+>
+> * Checked with: Unity 2018.3.14f1
+
+## Introduction
+
+The `Input Mutator ColliisonIgnorer` provides a way of allowing two or more objects to not interact with each other.
+
```

Comment:
Missing Prerequisites section, needs to read:

```
## Prerequisites

* [Install the Tilia.Mutators.CollisionIgnorer.Unity] package dependency in to your Unity project.
```

Also make sure to add the link to [Install the Tilia.Mutators.CollisionIgnorer.Unity] at the bottom of the guide.